src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -200,6 +206,13 @@ export default {
         },
         setEditState(isEditing) {
             this.isEditing = isEditing;
+            if (this.isEditing) {
```

Comment:
may wanna use `if (isEditing) {` instead to be safe, as `this.isEditing` ll get new value after `this.$nextTick(`

Reply:
Should not need to, $nextTick is used when you want to wait for the dom update before doing something. In this case we are only setting the internal isEditing value.

Reply:
ahh, right.